majifix311/src/test/java/com/example/majifix311/ReportServiceTest.java

```diff
@@ -124,4 +149,193 @@ void setMockEndpoint(String endpoint) {
 //            };
 //        }
     }
+
+    private TestObserver<TaggedProblemList> runTransformer(
```

Comment:
usually private classes should be at the end of the class, or in this case, after the `@Test` methods.

Reply:
Don't know that it makes a huge difference, but it's a private _method_. TestObserver is one of RxJava's standard classes.

Reply:
Don't know that it makes a huge difference, but it's a private method. TestObserver is one of RxJava's standard classes.

Reply:
Quite true -  I mistyped. It should be "...private *methods* should be..."

I noted this because I found myself going through this class in an order I'm not used to, and found it more challenging to find what I was looking for. I am used to having the tests/public methods at the top, as that's what is most important (what can I call/what is this trying to do/what is being tested). And the private below, if you need to dig into the details of how it is done. Not a big thing, but I expect if I interacted with this class, the first thing I would probably do is rearrange it to follow that convention. Sorry for my mistype. And, note, I sometimes do this too, especially during development. Organization is something I tend to clean up while reviewing my code before pushing.